src/isolate-duplicates/index.js

```diff
@@ -1,3 +1,38 @@
-function isolateDuplicates(text) {}
+function isolateDuplicates(text) {
+    let str = "";
+    let fullStr = "";
+    let count = 0;
+    array2 = [];
+    if (!(typeof text === 'string')){
+        throw("Please enter a valid string");
+    }
+    for (let i = 0; i < text.length; i++) {
+        if(i + 1 < text.length) {
+            if ( ( text.indexOf((text[i]).toUpperCase()) === text.indexOf((text[i+1]).toUpperCase())  ) && ( text.indexOf((text[i]).toLowerCase()) === text.indexOf((text[i+1]).toLowerCase()) ) ){
```

Comment:
The formatting here could be better. This conditional is complicated, it could be simplified.